### PR TITLE
Fix documentation on softmax_cross_entropy_with_logits_v2

### PR DESCRIPTION
**Pull request by mbrio**
- Return documentation was incorrectly stating shape of tensor

```
import tensorflow as tf
import numpy as np

logits = np.array([
    [5.0,  2.3, 0.0, -3.2, -0.8],
    [2.3,  0.1, 1.8,  1.1,  1.2],
    [-1.2, 1.1, 0.1,  2.5,  0.7]
])

labels = np.array([
    [1., 1., 0., 0., 0.],
    [0., 0., 1., 0., 1.],
    [0., 0., 0., 1., 0.]
])

x_entropy = tf.nn.softmax_cross_entropy_with_logits_v2(labels=labels, logits=logits)

with tf.Session() as sess:
    print(sess.run(tf.shape(labels)))  # [3, 5]
    print(sess.run(tf.shape(x_entropy)))  # [3]
```

In this example we see that the `x_entropy` shape does not equal the shape of `labels` as stated in the original documentation, it is of shape `[batch_size]`.

### CODE REVIEW

**Comment by mbrio** [comment]
I signed it!

File: tensorflow/python/ops/nn_ops.py [hunk 1]
```diff
@@ -1836,7 +1836,7 @@ def softmax_cross_entropy_with_logits_v2(
     name: A name for the operation (optional).
 
   Returns:
-    A `Tensor` of the same shape as `labels` and of the same type as `logits`
+    A `Tensor` of shape `[batch_size]` and of the same type as `logits`
```
Comment (reedwm) [inline tensorflow/python/ops/nn_ops.py]: This is only true if labels has rank 2 (although this statement is certainly more accurate than before this PR). If labels has a a higher rank than 2, e.g. if labels is of shape [d0, d1, ..., dn, num_classes], the output will have shape [d0, d1, ..., dn]. Can you clarify this?

Comment (reedwm) [inline tensorflow/python/ops/nn_ops.py]: Maybe also change the docstring for the softmax_cross_entropy_with_logits() function below this one.

**Review by reedwm — CHANGES_REQUESTED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 21 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 36 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 51 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by drpngx** [comment]
@mbrio could you pull rebase and push again? Any time to address changes?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
